sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusManagementClientImplBuilder.java

```diff
@@ -63,9 +64,18 @@ public ServiceBusManagementClientImplBuilder pipeline(HttpPipeline pipeline) {
         return this;
     }
 
+    /*
+     * The serializer to serialize an object into a string
+     */
     private SerializerAdapter serializerAdapter;
 
-    public ServiceBusManagementClientImplBuilder serializer(SerializerAdapter serializerAdapter) {
+    /**
+     * Sets The serializer to serialize an object into a string.
+     *
+     * @param serializerAdapter the serializerAdapter value.
+     * @return the ServiceBusManagementClientImplBuilder.
+     */
+    public ServiceBusManagementClientImplBuilder serializerAdapter(SerializerAdapter serializerAdapter) {
         this.serializerAdapter = serializerAdapter;
```

Comment:
`serializerAdapter` : Is this the name going to be in all the SDK as well ? This API name should be common across all.

Reply:
This is autogenerated by AutoRest.